server/test/services/applicant/ApplicantServiceTest.java

```diff
@@ -31,17 +31,12 @@
 import services.LocalizedStrings;
 import services.Path;
 import services.applicant.ApplicantService.ApplicantProgramData;
-import services.applicant.exception.ApplicantNotFoundException;
-import services.applicant.exception.ApplicationOutOfDateException;
-import services.applicant.exception.ApplicationSubmissionException;
-import services.applicant.exception.ProgramBlockNotFoundException;
+import services.applicant.exception.*;
```

Comment:
issue (blocking): no wildcard imports

Reply:
done.  My IntelliJ is auto switching to * imports on the 5th in a package it seems.... Will have to figure out how to disable that.